core/templates/components/summary-tile/exploration-summary-tile.component.ts

```diff
@@ -137,6 +141,14 @@ export class ExplorationSummaryTileComponent implements OnInit, OnDestroy {
     this.relativeLastUpdatedDateTime = this.getRelativeLastUpdatedDateTime();
     this.avgRating = this.getAverageRating();
     this.thumbnailIcon = this.getCompleteThumbnailIconUrl();
+    this.explTitleTranslationKey = (
+      this.i18nLanguageCodeService.getExplorationTranslationKey(
+        this.explorationId, TranslationKeyType.TITLE)
+    );
+    this.objectiveTranslationKey = (
```

Comment:
If you're calling the above "exploration title" then this should be "exploration objective", for parity. 

The same principle applies elsewhere.

Reply:
Done!